src/commands.rs

```diff
@@ -339,6 +339,22 @@ pub enum OfflineWalletSubCommand {
         /// Selects which policy should be used to satisfy the internal descriptor
         #[structopt(name = "INT_POLICY", long = "internal_policy")]
         internal_policy: Option<String>,
+        /// Optionally create an OP_RETURN output containing given String.
+        #[structopt(
+            name = "ADD_STRING",
+            long = "add_string",
+            short = "s",
+            conflicts_with = "ADD_DATA"
```

Comment:
I feel like there is no need to restrict to only one variation in a single command. Each `tx_builder.add_data()` call adds a new 0 value output to the transaction. So we can have multiple OP_RETURNS of both varieties in a single `create_tx` call. 

I can't think of any reason not to have that..

Reply:
Yes Raj I agree, maybe we shouldn't restrict both varitieties of `tx_builder.add_data() ` in a single create_x call like you mentioned. Steve @notmandatory  what do you think?

Reply:
I hadn't considered multiple op_returns. My suggestion is to do that as a separate PR after this one is merged since it will require new logic and testing, and having multip op_returns won't invalidate any of the functionality already here only expand it.